src/network/rpc/TestClient.h

```diff
@@ -2,235 +2,260 @@
  * This file is generated by jsonrpcstub, DO NOT CHANGE IT MANUALLY!
  */
 
-#pragma once
+#ifndef JSONRPC_CPP_STUB_TARAXA_NET_TESTCLIENT_H_
+#define JSONRPC_CPP_STUB_TARAXA_NET_TESTCLIENT_H_
 
```

Comment:
I guess these guards were generated automatically ? As we are using pragmas but this seems like generation-related issue...

Reply:
this is a generated code

Reply:
@JakubFornadel please resolve - can't influence the generator